### PR TITLE
TS 1009 Payment Module Automation Test 

### PR DESCRIPTION
**Pull request by Ozleblebici**
image
 
Ready for Review

### CODE REVIEW

**Review by Filizbagdat — APPROVED** [review]

File: src/PaymentTest.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+public class PaymentTest {
+
+    public static void main(String[] args) {
+        System.out.println("Payment test1");
+        System.out.println("Payment test3");
+        System.out.println("Payment test4");
```
Comment (CevahirBb) [inline src/PaymentTest.java:6]: hi

Comment (PatelSarita) [inline src/PaymentTest.java:6]: Looks like you have to add some more code

**Review by PatelSarita — CHANGES_REQUESTED** [review]

**Review by RosanaKmetovikj — CHANGES_REQUESTED** [review]
need to change line 4

**Review by CevahirBb — COMMENTED** [review]
can you add test2

File: src/PaymentTest.java (hunk 1 above)
Comment (Ozleblebici) [reply src/PaymentTest.java:6]: no thank you

Comment (Ozleblebici) [reply src/PaymentTest.java:6]: hello

**Review by RosanaKmetovikj — APPROVED** [review]

**Review by PatelSarita — APPROVED** [review]

**Pull request merged**